data/packages/mjirv/dbt_datamocktool/versions/0.1.0-beta-fishtown.json

```diff
@@ -0,0 +1,25 @@
+{
+    "id": "mjirv/dbt_datamocktool/0.1.0-beta-fishtown",
+    "name": "dbt_datamocktool",
+    "version": "0.1.0-beta-fishtown",
```

Comment:
Hey there @mjirv 👋   The package hub only supports[ semantic versioning ](https://semver.org/ )at this time and the -beta and -fishtown labels in the version field are tripping up the tests here. 

Would you mind taking a look and adjusting the version names? We'll then be able to include these on the Hub.

Reply:
Hi @annafil! Thanks for taking a look at this. I’ll adjust and comment here when done.

One question: my understanding is those labels _are_ part of the semantic versioning spec for pre-releases (see https://semver.org/#spec-item-9).

Are prerelease versions not supported here, or am I doing something wrong?

Mostly asking because I’d prefer to leave a version available that’s compatible with `fishtown-analytics/dbt-utils` for now (though I I can point people to a GitHub branch if not supported).

Reply:
@annafil I updated the version names, so this should work now!

Reply:
Suhweeet that did the trick!! Nice job @mjirv 🙏